contents/articles/roll-call-brett/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Roll call: Brett Langdon"
+author: brett-langdon
+date: 2015-06-21
+template: article.jade
+---
+
+Continuation of the series to look behind the curtain into the engineering team at [Underdog.io][]
+
+[Underdog.io]: https://underdog.io
+
+---
+
+<img alt="Photo of Brett" src="brett.jpg" style="width: auto; float: left; padding-right: 10px;" />
+
+#### Who are you and what do you do at Underdog.io?
+My name is Brett Langdon and I am senior software engineer at [Underdog.io][].
+
+Since we are still a small team, we all get to wear many hats and get our hands dirty with every aspect of our platform. Some days I am either writing backend Python or Node.js, writing frontend HTML/CSS/JS or staring at either our [AWS][] management console or [Datadog][] dashboards.
+
+[AWS]: https://aws.amazon.com/
+[Datadog]: http://www.datadoghq.com/
+
+[Underdog.io]: https://underdog.io
+
+#### What hardware do you use?
+
+I use a 21" iMac and a WASD [CODE keyboard][] to help ensure the entire office knows when I am typing something.
+
+[CODE keyboard]: https://codekeyboards.com/
+
+#### What software do you use?
+
+I spend a good majority of my day in the terminal, so for productivity there I use [iterm2][] as a terminal emulator, [oh-my-zsh][] to provide zsh plugins and themes, [homebrew][] for installing third party tools and I use [tmux][] to organize my terminal workspace.
+
+Since I am a software engineer I have to spend a lot of time writing code, for that I use [emacs][] along with some fun packages like [elpy][] to help auto-complete and lint my code while I am writing it.
+
+When I am not writing code, I am usually running it. A few python packages that I do not think I could live without are the [bpython][] interpreter, [pdb++][] and [virtualenvwrapper][].
+
+Lastly, when I am not writing or running code it is usually spent sending fellow team members appropriate emojis and gifs based on the context of some other conversation. For communication at work we use [slack][].
```

Comment:
There should prob be a comma after "running code" to indicate that we broke out of the "or"